.github/workflows/main.yml

```diff
@@ -29,9 +29,13 @@ jobs:
         run: sudo apt-get install autoconf automake bison build-essential clang doxygen flex g++ git libffi-dev libncurses5-dev libtool libsqlite3-dev make mcpp python sqlite zlib1g-dev
 
       # Runs a single command using the runners shell
-      - name: Run a one-line script
+      - name: Run all tests
         run: bazel test //src/...
 
       # Verifies that the Arcs parser works
       - name: Verify that Arcs parser works.
         run: bazel build //third_party/arcs/examples:consume
+
+      # Verifies that Arcs manifest tests that we do not yet handle parse correctly.
```

Comment:
Just an observation. I think it is time to set up something more performant for tests. It is already taking up to 17m per tests.

Reply:
Acknowledged.